projects/knora-ui/src/lib/viewer/values/date-value/calendar-header/calendar-header.component.ts

```diff
@@ -0,0 +1,90 @@
+/** Custom header component containing a calendar format switcher */
+import {JDNConvertibleCalendarDateAdapter} from 'jdnconvertiblecalendardateadapter';
+import {FormBuilder, FormControl, FormGroup, Validators} from '@angular/forms';
+import {JDNConvertibleCalendar} from 'jdnconvertiblecalendar';
+import {MatCalendar, MatDatepickerContent} from '@angular/material/datepicker';
+import {DateAdapter} from '@angular/material/core';
+import {Component, Host, Inject, OnDestroy, OnInit} from '@angular/core';
+import {Subscription} from "rxjs";
+
+@Component({
+  selector: 'kui-calendar-header',
+  template: `
```

Comment:
I think you should move this to the template file to keep it consistent with the way we've been creating all the other components

Reply:
done in https://github.com/dasch-swiss/knora-ui-ng-lib/pull/35/commits/d7e2127c96a9da46b65ef6757fc09cb616393cef